tst/EnergyPlus/unit/SizingAnalysisObjects.unit.cc

```diff
@@ -116,9 +116,8 @@ class SizingAnalysisObjectsTest : public testing::Test
         Environment(4).DesignDayNum = 2;
         Environment(4).SeedEnvrnNum = 2;
 
-        TimeValue.allocate(2);
-        TimeValue(1).TimeStep >>= TimeStepZone;
-        TimeValue(2).TimeStep >>= DataHVACGlobals::TimeStepSys;
+        OutputProcessor::SetupTimePointers("ZONE", TimeStepZone);
+        OutputProcessor::SetupTimePointers("HVAC", DataHVACGlobals::TimeStepSys);
```

Comment:
This looks weird without the DataHVACGlobals namespace on just one of them.

Reply:
It was like that, but yeah I should probably have resolved the namespace. TimeStepZone isn't in DataHVACGlobals, it's in DataGlobals.